plenario/api/condition_builder.py

```diff
@@ -0,0 +1,162 @@
+from sqlalchemy import and_, or_, func
+
+
+# field_ops
+# =========
+# Map codes we accept in API docs to SQLAlchemy function names
+
+field_ops = {
+    'gt': '__gt__',
+    'ge': '__ge__',
+    'lt': '__lt__',
+    'le': '__le__',
+    'ne': '__ne__',
+    'like': 'like',
+    'ilike': 'ilike',
+    'is': 'is_',
+    'isnot': 'isnot',
+    'eq': 'eq',
+    'in': 'in'
+}
+
+
+class ConditionBuilder(object):
```

Comment:
I think I originally had ConditionBuilder objects carrying some sort conditions list around inside, but I eventually scrapped that. Yep those methods definitely don't have to be within a class